core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/login/LoginModifierConfig.java

```diff
@@ -36,11 +37,11 @@
 
   private static final String DEFAULT_LOGIN_TOKEN = "login-token";
 
-  private static final String LOGIN_PARAM = "login";
+  static final String LOGIN_PARAM = "login";
```

Comment:
I'm not sure I'm reading this right but if this is literally going to be the name of the Environment variable to set, how about prefixing it with something more explicit?At the moment, it's quite generic and not immediately associated with AET, a kind of thing that might get users and administrators confusing.

Reply:
This constant is the name of a param that is passed via XML suite definition. The actual environment variable name is configured through that param, see the example:

```xml
<login login="${LOGIN_EXAMPLE_ADMIN_PANEL}"
                password="${PASS_EXAMPLE_ADMIN_PANEL}"
                login-page="/login.html"
                login-input-selector="//input[@name='j_username']"
                password-input-selector="//input[@name='j_password']"
                submit-button-selector="//*[@type='submit']" />
```